autobotAI_integrations/integrations/crowdstrike/__init__.py

```
@@ -126,3 +167,56 @@ def generate_steampipe_creds(self) -> SteampipeCreds:
+
+    def generate_python_sdk_creds(self) -> SDKCreds:
+        return SDKCreds(
+            envs={
+                "FALCON_CLIENT_ID": self.integration.client_id,
+                "FALCON_CLIENT_SECRET": self.integration.client_secret,
+                "FALCON_CLOUD": self.integration.client_cloud,
+            }
+        )
+
+    def build_python_exec_combinations_hook(
+        self, payload_task: PayloadTask, client_definitions: List[SDKClient]
```

Ruff (0.15.20)

[warning] Unused method argument: `client_definitions`

(ARG002)